### DISCUSSION TITLE
Open an .xlsm file in vsc

### DISCUSSION THREAD

**Discussion by JohnDoedz** [serkonda7/vscode-vba#97] [Q&A] [answered]
Hi guys,
May I know how to edit my .xlsm files in VSC?
I installed the "VBA" extension
Thanks in advance

**Comment by serkonda7** [comment]
You cannot edit .xlsm files using this extension. However an exporter for macros from xlsm is in it's very early stages of development.

**Reply by JohnDoedz** [reply]
Thank you.

**Reply by fvandonk** [reply, marked as answer]
Xlwings created a  free plugin for VSC that will allow you do edit the VBA code in your .XLSM files. I have been doing this successfully for a while now. 
Check out this YouTube video that show you how to do this and where to download the install files. 

https://www.youtube.com/watch?v=xoO-Fx0fTpM&ab_channel=xlwings

The link can be found in the description of the video.
